### PR TITLE
Clean up MaaS and local scripts for Flake8 (kilo backport)

### PR DESCRIPTION
**Pull request by nrb**
Fix all reported hacking errors

Addresses #240 
Addresses #272
(cherry picked from commit e00db31c320e22627282c384eca6acd2d600040c)

### CODE REVIEW

File: maas/maas_common.py [hunk 1]
```diff
@@ -71,7 +72,7 @@ def get_cinder_client(previous_tries=0):
         return cinder
 
 try:
-    from glanceclient import Client as g_client
+    import glanceclient as g_client
```
Comment (d34dh0r53) [inline maas/maas_common.py]: Little nit here, later on line 121 we're importing novaclient.client as nova_client, we should probably keep with that convention and call this glance_client.

File: maas/maas_common.py [hunk 2]
```diff
@@ -169,8 +171,8 @@ def get_nova_client(auth_token=None, bypass_url=None, previous_tries=0):
         return nova
 
 try:
-    from keystoneclient.v3 import client as k_client
     from keystoneclient.openstack.common.apiclient import exceptions as k_exc
+    from keystoneclient.v3 import client as k_client
```
Comment (d34dh0r53) [inline maas/maas_common.py]: Same as above, I'd like to see this called keystone_client.

File: maas/maas_common.py [hunk 3]
```diff
@@ -231,8 +233,8 @@ def get_keystone_client(auth_ref=None, endpoint=None, previous_tries=0):
 
 
 try:
-    from neutronclient.neutron import client as n_client
     from neutronclient.common import exceptions as n_exc
+    from neutronclient.neutron import client as n_client
```
Comment (d34dh0r53) [inline maas/maas_common.py]: Change to neutron_client

**Comment by d34dh0r53** [comment]
A couple of nits but otherwise looks good, I'll give this a +1 and ignore my nits if this is time sensitive.

**Comment by nrb** [comment]
@d34dh0r53 While I agree with you, this is also a backport, so I'm hesitant to make the change in this PR. We can file a clean up issue for renaming the clients, though.

**Comment by Apsu** [comment]
:+1:

**Comment by d34dh0r53** [comment]
:+1:

**Pull request merged**